lib/api/controllers/documentController.js

```diff
@@ -37,101 +65,101 @@ function DocumentController (kuzzle) {
     }
 
     if (request.input.args.size) {
-      let documentsCount = request.input.args.size - (request.input.args.from || 0);
+      const documentsCount = request.input.args.size - (request.input.args.from || 0);
 
-      if (documentsCount > kuzzle.config.limits.documentsFetchCount) {
-        throw new SizeLimitError(`document:search cannot fetch more documents than the server configured limit (${kuzzle.config.limits.documentsFetchCount})`);
+      if (documentsCount > this.kuzzle.config.limits.documentsFetchCount) {
+        throw new SizeLimitError(`document:search cannot fetch more documents than the server configured limit (${this.kuzzle.config.limits.documentsFetchCount})`);
       }
     }
 
-    return engine.search(request);
-  };
+    return this.engine.search(request);
+  }
 
   /**
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.scroll = function documentScroll (request) {
+  scroll(request) {
     if (!request.input.args.scrollId) {
       throw new BadRequestError('document:scroll must specify a "scrollId" argument');
     }
 
-    return engine.scroll(request);
-  };
+    return this.engine.scroll(request);
+  }
 
   /**
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.get = function documentGet (request) {
+  get(request) {
     assertHasId(request);
     assertHasIndexAndCollection(request);
 
-    return engine.get(request);
-  };
+    return this.engine.get(request);
+  }
 
   /**
    * Get specific documents according to given ids
    *
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.mGet = function documentMGet (request) {
+  mGet(request) {
     assertHasBody(request);
     assertBodyHasAttribute(request, 'ids');
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
-    if (request.input.body.ids.length > kuzzle.config.limits.documentsFetchCount) {
-      throw new SizeLimitError(`Number of gets to perform exceeds the server configured value (${kuzzle.config.limits.documentsFetchCount})`);
+    if (request.input.body.ids.length > this.kuzzle.config.limits.documentsFetchCount) {
+      throw new SizeLimitError(`Number of gets to perform exceeds the server configured value (${this.kuzzle.config.limits.documentsFetchCount})`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 147 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)